english/dog-eat/insulin/insulin.md

```diff
@@ -0,0 +1,5 @@
+You decide to go to the shops and fetch some chocolate and graham crackers, because, well, the dog has had some marshmellows and you'd like some too. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.